cluster/cluster.go

```diff
@@ -81,7 +81,17 @@ func NumOfDaemons() int {
 
 // Start a local cluster of gubernator servers
 func Start(numInstances int) error {
-	peers := make([]gubernator.PeerInfo, numInstances, numInstances)
+	// Ideally we should let the socket choose the port, but then
+	// some things like the logger will not be set correctly.
+	var peers []gubernator.PeerInfo
```

Comment:
I think we should leave the original `make()` version to avoid a chunk of allocations